RecoParticleFlow/PFProducer/python/ecalBarrelClusterFastTimer_cfi.py

```diff
@@ -0,0 +1,12 @@
+import FWCore.ParameterSet.Config as cms
+
+ecalBarrelClusterFastTimer = cms.EDProducer(
```

Comment:
it looks like this file does not really belong in here and fits better with #15613